package/MDAnalysis/coordinates/XDR.py

```diff
@@ -116,8 +116,12 @@ def _read_offsets(self, store=False):
                 np.savez(offsets_filename(self.filename),
                          offsets=offsets, size=size, ctime=ctime)
             except Exception as e:
-                warnings.warn("Couldn't save offsets because: {}".format(
-                    e.message))
+                try:
```

Comment:
Did the exception format change in Py3? Is there something in `six` so that we don't have to write the try block everytime? How do other packages deal with this?